unidist/core/backends/mpi/core/object_store.py

```diff
@@ -0,0 +1,84 @@
+from unidist.core.backends.mpi.core.local_object_store import LocalObjectStore
```

Comment:
put a licence header and one line string about file content.

Reply:
done